### ISSUE TITLE
Batch normalization in pytorch

### ISSUE THREAD

**Issue by zorrocai** [lberrada/bn.pytorch#1]
Hey~, have you written batch normalization in pytorch successfully?
